htdocs/class/file/folder.php

```diff
@@ -286,7 +286,7 @@ public function _findRecursive($pattern, $sort = false)
      * @access public
      * @static
      */
-    public function isWindowsPath($path)
+    public static function isWindowsPath($path)
```

Comment:
I can't see any good reason for these changed methods to be declared static. The callers all should use $this-> instead.